source/css/sass/mixins/_sizes.scss

```diff
@@ -0,0 +1,25 @@
+@import "../functions/sizes";
+
+@mixin height($size_in_px) {
+  height: #{$size_in_px}px;
+  height: #{get-rem-from-px($size_in_px)}rem;
+}
+
+@mixin width($size_in_px) {
+  width: #{$size_in_px}px;
+  width: #{get-rem-from-px($size_in_px)}rem;
+}
+
+@mixin constrain-height($size_in_px, $extreme) {
```

Comment:
We should look at redoing `constrain-width` and `constrain-height` as  something like `constrain-inline-size` and `constrain-block-size` to match our sizing approach elsewhere.